src/components/Article/Article.jsx

```diff
@@ -3,24 +3,51 @@ import { connect } from 'react-redux';
 import { Parser as HtmlToReact } from 'html-to-react';
 import { PropTypes } from 'prop-types';
 import moment from 'moment';
+import {
+  Editor, EditorState, convertFromRaw, CompositeDecorator,
+} from 'draft-js';
+import MultiDecorator from 'draft-js-plugins-editor/lib/Editor/MultiDecorator';
 import { fetchArticle, likeArticle, dislikeArticle } from '../../redux/actions/articleActions';
+import { mediaBlockRenderer } from '../../helpers/editorPlugins/mediaBlockRenderer';
+import addLinkPlugin from '../../helpers/editorPlugins/addLink';
+import createHighlightPlugin from '../../helpers/editorPlugins/highlight';
 import { onUserRateArticle, setNextPath } from '../../redux/actions/currentUserActions';
 
+const highlightPlugin = createHighlightPlugin();
+
 export class Article extends Component {
+  decorator = new MultiDecorator([new CompositeDecorator(addLinkPlugin.decorators)]);
+
   componentDidMount() {
     const {
       match: {
         params: { articleSlug },
       },
       getArticle,
     } = this.props;
+
+    console.log(this.props);
```

Comment:
🤣 .. 😠 ATTENTION TO DETAIL. REVIEW YOUR OWN PR. @mcaleb808 @filschristian @Nziranziza You will again pay for this.

Reply:
@kimotho-njoki I am guilty of this! I forget to remove `console.log`, It is removed now